src/main/java/com/hedvig/backoffice/services/product_pricing/dto/ManualRedeemCampaignRequest.kt

```diff
@@ -0,0 +1,8 @@
+package com.hedvig.backoffice.services.product_pricing.dto
+
+import java.time.LocalDate
+
+data class ManualRedeemCampaignRequest (
+  val campaignCode: String,
+  val activationDate: LocalDate
```

Comment:
`activationDate` is nullable and send it as null if the hope operator is not specifically set in it to something
```suggestion
  val activationDate: LocalDate?
```